config/locales/en.yml

```diff
@@ -368,14 +368,17 @@ en:
     edit:
       page_title: >
         Edit a submission
+    index:
```

Comment:
The page title for the [All Teams page](https://nusskylab-dev.comp.nus.edu.sg/submissions) was
earlier translation missing: en.submissions.index.page_title.